content/python/concepts/collections-module/terms/deque/deque.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'collections.deque()'
+Description: 'A method that creates a deque object'
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+Tags: 
+  - 'Data Structures'
+  - 'Data Types'
+  - 'Queues'
+  - 'Stacks'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`deque()`** function creates a new `deque` object.
+
+## Syntax
+
+```pseudo
+my_deque = deque(iterable, maxlen)
+```
+`maxlen` is optional and is used to limit the size of the `deque`.
+
+## Example 1
```

Comment:
Instead of creating separate headings lets put all this under one heading as ## Example. Use the bold ** ** for the sub headings. Also, use the shell tag and create output blocks for the example snippets, and don't create output blocks for the last code byte code snippet, Thank you.